### PR TITLE
EVG-17134: Expose container configurations to production

### PR DESCRIPTION
**Pull request by hadjri**
EVG-17134

### Description
Now that we have multiple teams moving forward to onboard containers to their project,  we should allow for the exposure of the containers tab in Spruce. Given that only a select few beta testers are using this, and they are manually coordinating with us, I added a banner to make that clear.

image

### CODE REVIEW

File: src/pages/projectSettings/tabs/ContainersTab/ContainersTab.tsx [hunk 1]
```diff
@@ -31,12 +32,20 @@ export const ContainersTab: React.VFC<TabProps> = ({
...
   return (
-    <SpruceForm
-      fields={fields}
-      formData={formData}
-      onChange={onChange}
-      schema={schema}
-      uiSchema={uiSchema}
-    />
+    <>
+      <Banner data-cy="disabled-webhook-banner" variant="warning">
```
Comment (khelif96) [inline src/pages/projectSettings/tabs/ContainersTab/ContainersTab.tsx:36]: The `data-cy` is not necessary since we aren't using it for any tests.

**Review by khelif96 — APPROVED** [review]
LGTM!

**Comment by ablack12** [comment]
to verify, this banner only displays on the container page right? (love the idea also!)

**Comment by hadjri** [comment]
> to verify, this banner only displays on the container page right? (love the idea also!)

Yep!

**Pull request merged**